cmd/command/login.go

```diff
@@ -15,8 +16,11 @@ import (
 )
 
 var Login = cli.Command{
-	Name:   "login",
-	Usage:  "Log in to Common Fate",
+	Name:  "login",
+	Usage: "Log in to Common Fate",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{Name: "lazy", Usage: "the lazy flag lets granted decide whether a new login flow should be initiated based on the token expiry", Aliases: []string{"l"}},
```

Comment:
I don't think this will be used enough outside of scripts to warrant a short alias

```suggestion
		&cli.BoolFlag{Name: "lazy", Usage: "the lazy flag lets granted decide whether a new login flow should be initiated based on the token expiry"},
```